drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/utils/GitHubLastUpdateCache.java

```diff
@@ -0,0 +1,106 @@
+package org.jboss.arquillian.drone.webdriver.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.jboss.arquillian.drone.webdriver.utils.Constants.DRONE_TARGET_DIRECTORY;
+
+public class GitHubLastUpdateCache {
+
+    private static final String ASSET_PROPERTY = "asset";
+    private static final String LAST_MODIFIED_PROPERTY = "lastModified";
+
+    private final File cacheDirectory;
+    private final Gson gson;
+
+    public GitHubLastUpdateCache(final File cacheDirectory) {
+        this.gson = new GsonBuilder().registerTypeAdapter(new TypeToken<ZonedDateTime>(){}.getType(), new ZonedDateTimeConverter()).create();
+        this.cacheDirectory = cacheDirectory;
+    }
+
+    public GitHubLastUpdateCache() {
+        this(new File(DRONE_TARGET_DIRECTORY + File.separator + "gh_cache" + File.separator));
```

Comment:
Ok, then I have missed it in the code. But I can recall there are some files downloaded there. Maybe for test purposes then.